[2025-05-14T05:34:27.418+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_data_ingestion.wait_for_olist_geolocation_dataset manual__2025-05-14T05:34:14.952420+00:00 [queued]>
[2025-05-14T05:34:27.444+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_data_ingestion.wait_for_olist_geolocation_dataset manual__2025-05-14T05:34:14.952420+00:00 [queued]>
[2025-05-14T05:34:27.446+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-14T05:34:27.673+0000] {taskinstance.py:2191} INFO - Executing <Task(FileSensor): wait_for_olist_geolocation_dataset> on 2025-05-14 05:34:14.952420+00:00
[2025-05-14T05:34:27.697+0000] {standard_task_runner.py:60} INFO - Started process 109 to run task
[2025-05-14T05:34:27.712+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_data_ingestion', 'wait_for_olist_geolocation_dataset', 'manual__2025-05-14T05:34:14.952420+00:00', '--job-id', '156', '--raw', '--subdir', 'DAGS_FOLDER/ecommerce_etl_dag.py', '--cfg-path', '/tmp/tmpi1pk9jmi']
[2025-05-14T05:34:27.729+0000] {standard_task_runner.py:88} INFO - Job 156: Subtask wait_for_olist_geolocation_dataset
[2025-05-14T05:34:27.885+0000] {task_command.py:423} INFO - Running <TaskInstance: ecommerce_data_ingestion.wait_for_olist_geolocation_dataset manual__2025-05-14T05:34:14.952420+00:00 [running]> on host 770089f1497a
[2025-05-14T05:34:28.116+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ecommerce_data_ingestion' AIRFLOW_CTX_TASK_ID='wait_for_olist_geolocation_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-05-14T05:34:14.952420+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-14T05:34:14.952420+00:00'
[2025-05-14T05:34:28.182+0000] {base.py:83} INFO - Using connection ID 'fs_default' for task execution.
[2025-05-14T05:34:28.186+0000] {filesystem.py:66} INFO - Poking for file /opt/***/data/extracted_data/olist_geolocation_dataset.csv
[2025-05-14T05:34:28.189+0000] {filesystem.py:71} INFO - Found File /opt/***/data/extracted_data/olist_geolocation_dataset.csv last modified: 20211001132344
[2025-05-14T05:34:28.189+0000] {base.py:295} INFO - Success criteria met. Exiting.
[2025-05-14T05:34:28.208+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ecommerce_data_ingestion, task_id=wait_for_olist_geolocation_dataset, execution_date=20250514T053414, start_date=20250514T053427, end_date=20250514T053428
[2025-05-14T05:34:28.316+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-14T05:34:28.445+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
